zerver/data_import/slack.py

```diff
@@ -823,10 +824,14 @@ def get_messages_iterator(
                     # skipping those messages is simpler.
                     continue
                 if message.get("mimetype") == "application/vnd.slack-docs":
-                    # This is a Slack "Post" which is HTML-formatted,
-                    # and we don't have a clean way to import at the
-                    # moment.  We skip them on import.
-                    continue
+                    # This is a Slack "Post" which is HTML-formatted
+                    file_url = message.get("url_private_download")
+                    response = requests.get(file_url)
+                    html_content = str(response.content, encoding="utf-8")
```

Comment:
`response.text` should do. According to https://requests.readthedocs.io/en/latest/user/quickstart/#response-content, 
> Requests will automatically decode content from the server. Most unicode charsets are seamlessly decoded.

Reply:
Yes I made the changes and it seems to be working. However I had to use `response.encoding = "utf-8"` before as simply using `response.text` only was returning extra special characters.